### DISCUSSION TITLE
how can i add yaml file to my cluster using kuberentes java client

### DISCUSSION THREAD

**Discussion by Elias-Swais** [kedacore/keda#4891] [Q&A / Need Help]
im trying to add this yaml file :
kind: HTTPScaledObject
apiVersion: http.keda.sh/v1alpha1
metadata:
    name: nginx-http-scaledobject
spec:
    host: myhost.com
    targetPendingRequests: 1
    scaleTargetRef:
        deployment: nginx-deployment
        service: nginx-deployment
        port: 80
    replicas:
        min: 0
        max: 3
        
to my kuberentes cluster , im using java and is there any way to use the java client to add this to my cluster

**Comment by JorTurFer** [comment]
Hi,
I think that this isn't related with KEDA, you should check java client docs to check how you can create dynamic manifests. In java you won't have the api deifnition, but kubernetes client usually has a dynamic client that you can use. IDK if java has it but as I said, you should check in the client docs/repo
